src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -28,7 +28,7 @@
   public static final String HDFS_BINARY_DIR = "hdfs";
 
   // Current HDFS Binary File Name
-  public static final String HDFS_BINARY_FILE_NAME = "hdfs-mesos-0.0.2.tgz";
+  public static final String HDFS_BINARY_FILE_NAME = "hdfs-mesos-1.0.0.tgz";
```

Comment:
Are these the only places we hardcode the version number? (looks like it based on my greps)
Would be great to reduce this to a single place, if possible. 3 is fine for now, and some of this may change with @kensipe's gradle builds.